### ISSUE TITLE
issue of --min-score

### ISSUE THREAD

**Issue by sjhxy91** [jiarong/VirSorter2#58]
hi dear jiarong et al,
there is a question about 'min-score'.
Can I use the score in output file 'final_viral_socre.tsv' to filter the min-score I want ? because I got all of score >0.5 's results when I run vs2 first time by the default --min-socre setting.
Thanks~~

**Comment by jiarong** [comment]
Yes, you can filter the score table and pick filtered sequences in `final-viral-combined.fa`. It's the same as rerun with a different `--min-score`.

Also, there is a standard protocol for viral identification if you have not noticed in the doc:
https://www.protocols.io/view/viral-sequence-identification-sop-with-virsorter2-btv8nn9w

**Comment by sjhxy91** [comment]
> Yes, you can filter the score table and pick filtered sequences in `final-viral-combined.fa`. It's the same as rerun with a different `--min-score`.
> 
> Also, there is a standard protocol for viral identification if you have not noticed in the doc:
> https://www.protocols.io/view/viral-sequence-identification-sop-with-virsorter2-btv8nn9w

Thanks again jiarong
